metadata_mapper/mappers/oai/chapman_mapper.py

```diff
@@ -1,29 +1,29 @@
-from typing import Union
+from typing import Union, Any
 
 from .oai_mapper import OaiRecord, OaiVernacular
+from ..mapper import Validator
 
 
 class ChapmanRecord(OaiRecord):
-    """Mapping discrepancies:
-
-        * `type` field for images contains "Image" in Solr, but "text" in mapped data
-
-    """
-
     def UCLDC_map(self):
         return {
             'description': self.map_description,
-            'identifier': self.map_identifier
+            'identifier': self.map_identifier,
+            "source": self.source_metadata.get("collection_name"),
```

Comment:
The mapping of `source` should be moved to the oai_mapper and should read:

`"source": self.source_metadata.get("source"),`